app/code/Magento/ImportExport/Test/Unit/Model/Import/Source/FileParser/_files/test_options.csv

```diff
@@ -0,0 +1 @@
+column1|;column2;|column3
```

Comment:
Would love to get an advice how to overcome a problem of running fast "integration" test witout bootstrapping all database and other components.